packages/protocol/contracts/bridge/libs/LibBridgeData.sol

```diff
@@ -44,6 +44,6 @@ library LibBridgeData {
     function hashMessage(
```

Comment:
Shall we also update the comment of this function?

Reply:
fixed